docs/products/postgresql/howto/pagila.rst

```diff
@@ -0,0 +1,258 @@
+Sample dataset: Pagila
+======================
+
+Aiven provides a sample database you can import in your Aiven for PostgreSQL service. This page covers information about the database and the procedure to get it up and running.
+
+Pagila is a PostgreSQL port of the `Sakila Sample Database <https://dev.mysql.com/doc/sakila/en/>`_. The examples here use one from ``devrimgunduz``, `version 2.1.0 <https://github.com/devrimgunduz/pagila>`_.
+
+Sakila (and thus, Pagila) is a database representing a DVD rental store (remember those?!), containing information about films (like title, category, actresses), rental stores (like address, staff members, customers) and rentals, where a customer rents a film from a store through its staff.
+
+With all these relational information, Pagila is a perfect fit to play around with PostgreSQL and the SQL language.
+
+Load Pagila on PostgreSQL
+-------------------------
+
+Before exploring the Pagila database, follow the :doc:`create new service article<../../../platform/howto/create_new_service>` to spin up a PostgreSQL instance.
+
```

Comment:
Should we add a couple of subtitles here? 
e.g. "Load Sample Dataset via button" vs "Load Sample Dataset manually"

When I initially read the article I got lost into thinking I had to do both.

Reply:
Done!